setup.py

```diff
@@ -75,6 +75,9 @@
         'global_settings = indico_zodbimport.modules.global_settings:GlobalSettingsImporter',
         'badges_posters = indico_zodbimport.modules.badges_posters:BadgePosterImporter',
         'event_data = indico_zodbimport.modules.event_data:EventDataImporter',
+    ],
+    'babel.extractors': [
+        'jinja2_indico = indico.util.translations:jinja2_babel_extract'
```

Comment:
You can omit that and add this to `babel.cfg` instead in the `[extractors]` section:

    jinja2_indico = indico.util.translations:jinja2_babel_extract